apps/admin-portal/src/components/roles/roles-search.tsx

```diff
@@ -181,19 +181,25 @@ export const RoleSearch: FunctionComponent<RoleSearchProps> = (props: RoleSearch
                                 hidden={ true }
                                 type="divider"
                             />
-                            <Form.Group inline={ true }>
-                                <Field
-                                    size="small"
-                                    type="submit"
-                                    value={ t("common:search").toString() }
-                                />
-                                <Field
-                                    className="link-button"
-                                    size="small"
-                                    type="reset"
-                                    value={ t("common:resetFilters").toString() }
-                                />
-                            </Form.Group>
+                            <Grid>
+                                <Grid.Row columns={ 1 }>
+                                    <Grid.Column width={ 3 }>
+                                        <Field
+                                            size="small"
+                                            type="submit"
+                                            value={ t("common:search").toString() }
```

Comment:
Why do we need the `toString()`. We already get a string value here right?

Reply:
Fixed with commit 2f2db4b